docs/pages/management/admin/self-signed-certs.mdx

```diff
@@ -0,0 +1,117 @@
+  [Certificate Authority](https://goteleport.com/blog/ssh-certificate-authorities-explained/)
+  to prevent a [Man-In-The-Middle (MITM)](https://en.wikipedia.org/wiki/Man-in-the-middle_attack) attack.
+
+  This guide is provided for the benefit of users who are interested in evaluating Teleport in a
+  non-production environment, without having to configure certificates.
+</Notice>
+
+## Prerequisites
+
+(!docs/pages/includes/edition-prereqs-tabs.mdx!)
+
+## Self-Signed Certs: `teleport`
+
+When running a Teleport service with `teleport`, if the service you are starting is configured to point to
+the Teleport Proxy Service endpoint and the proxy service is using self-signed certs, then `teleport` will need
+to be run with the `--insecure` flag to disable verification of the
+proxy's TLS certificate. This is the case when:
+- The Teleport config file `auth_servers` setting is set to the proxy endpoint:
+  - `auth_servers: ["tele.example.com:443"]` or
+  - `auth_servers: ["tele.example.com:3080"]`
+- Teleport is started with the `--auth-server` flag pointing to the proxy endpoint:
+  - `teleport [app | db] start --auth-server=tele.example.com:443` or
+  - `teleport [app | db] start --auth-server=tele.example.com:3080`
+
+The way you make your Teleport service run with TLS certificate verification disabled depends on how you are
+running it - via `teleport` directly from the cli, or via helm chart, or via systemd:
+<Tabs>
+
+    <TabItem label="Running teleport directly">
+    When running `teleport` from the command line, pass the `--insecure` flag to disable
+    TLS certificate validation. For example:
+    ```sh
+    $ teleport start -c /etc/teleport.yaml --insecure
+    $ teleport app start -c /etc/teleport.yaml --insecure
+    $ teleport db start -c /etc/teleport.yaml --insecure
+    ```
+    Without the `--insecure` flag, you will see an error message that looks like
+    `x509: “tele.example.com” certificate is not trusted`.
+    </TabItem>
+
+    <TabItem label="Running teleport via helm chart">
+    For deploying a cluster, set
+    [extraArgs](../../reference/helm-reference/teleport-cluster.mdx#extraargs)
+    to include the extra argument: `--insecure`.
+    
+    
+    For a Helm Kube Agent, set the 
+    [insecureSkipProxyTLSVerify
+    ](../../reference/helm-reference/teleport-kube-agent.mdx#insecureskipproxytlsverify)
+    flag to `true`.
+    </TabItem>
+
+    <TabItem label="Running teleport via systemd">
+    Edit the `systemd` Teleport unit file to include `--insecure` in the `ExecStart` line, for example:
+    
+    `ExecStart=/usr/local/bin/teleport start --pid-file=/run/teleport.pid --insecure`
+    
+    You also need to run `sudo systemctl daemon-reload` and then `sudo systemctl restart teleport.service`
+    for your change to take effect.
+    
+    See [create a systemd service](./daemon.mdx#step-23-create-and-configure-a-systemd-service) for
```

Comment:
Why are we avoiding links in guides? Links are informative, concise, keep the guide from veering off-topic, and provide a single source of truth. If I inline an explanation for how to setup a systemd unit file, then that info has to be kept in sync with the systemd guide. It's also more information than I'd like to include in this guide. If I omit the link and don't inline, then the reader who doesn't know how to setup systemd has to go find that guide themselves.

We should be cautious of being overzealous in avoiding links. In this case, I've provided the additional info a user needs to modify their systemd unit file, assuming they know what a unit file is and how systemd services work. If they don't know then I provide a link to our systemd guide. I think that's worth including as a "see more info" link, because anyone who knows what I'm talking about already won't click the link, and anyone who doesn't know can click the link.

Having typed up my argument for links, maybe I should rephrase the sentence around links like this: `If you are unfamiliar with X, you can learn about it here: <link>`

Reply:
The [style guide](https://goteleport.com/docs/contributing/documentation/style-guide/#expectations) advises against adding links to the body sections of how-to guides because the goal of these guides is to get the reader up and running with a particular setup as quickly as possible. 

If we want to include a link in a how-to guide, I think we should make it explicit what a reader should get out of the link, e.g., something concrete like a configuration value, JSON document to copy/paste, etc. If we're telling a reader to consult a link to learn more information, we're not providing them with a quick path toward completing a goal.

Reply:
> If we're telling a reader to consult a link to learn more information, we're not providing them with a quick path toward completing a goal.

I see, in that case I agree we should remove the link here. I can always put "further reading" links at the bottom of the page.